client/src/components/ActionColumn.js

```diff
@@ -27,6 +27,9 @@ const ActionColumn = props => {
         <Link to={`/add-document-translation/${id}`}>
           <span>Submit Translation</span>
         </Link>
+        <Button onClick={() => props.handleCancelDocumentClick(id)}>
```

Comment:
The server endpoint that you are targeting is translations/id where the id is the translations id. 

On this file (ActionColumn), the variable id represents the document id and not the translations id.

You need to call props.handleCancelTranslationClick() with the id of the translation